backend/integrations/discord/bot.py

```
@@ -44,71 +41,39 @@ async def on_ready(self):
+
+            # 🔥 Direct Toolkit Execution
+            toolkit = GitHubToolkit()
+            result = await toolkit.execute(message.content)
+
+            response_text = result.get("message", "No response generated.")
+
+            if thread:
+                for i in range(0, len(response_text), 2000):
+                    await thread.send(response_text[i:i+2000])
 
         except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)